packages/typir/src/kinds/function-kind.ts

```diff
@@ -267,7 +273,6 @@ export class FunctionKind implements Kind {
         };
 
         // register Validations for input arguments of function calls (must be done here to support overloaded functions)
-        // TODO what about the case, that multiple variants match?? after implicit conversion?!
```

Comment:
If all need implicit conversion this should be an error, should it not, as they are kind of peers and no decision can be made? If there is a direct match among them it is superior.

Reply:
As I reviewed commit-wise, there might be some outdated remarks.